packages/plexus/src/LayoutManager/getLayout.tsx

```diff
@@ -19,43 +17,69 @@ import viz from 'viz.js';
 import convPlain from './dot/convPlain';
 import toDot from './dot/toDot';
 
-import type { LayoutOptions } from './types';
-import type { Edge, LayoutVertex, SizeVertex } from '../types/layout';
+import { EWorkerPhase, TLayoutOptions, TWorkerOutputMessage } from './types';
+import { TEdge, TLayoutVertex, TSizeVertex } from '../types/layout';
+
+enum EValidity {
+  Ok = 'Ok',
+  Warn = 'Warn',
+  Error = 'Error',
+}
+
+type TValidityError = {
+  validity: EValidity.Error;
+  message: string;
+};
+
+type TValidityOk = {
+  validity: EValidity.Ok;
+  message: void;
+};
+
+type TValidityWarn = {
+  validity: EValidity.Warn;
+  message: string;
+};
+
+type TVerticesValidity = TValidityError | TValidityOk | TValidityWarn;
 
 const SHIFT_THRESHOLD = 0.015;
 const VALIDITY_OK = 'VALIDITY_OK';
-const VALIDITY_ERROR = 'VALIDITY_ERROR';
 const VALIDITY_WARN = 'VALIDITY_WARN';
+const VALIDITY_ERROR = 'VALIDITY_ERROR';
```

Comment:
not _that_ important but seems like this should've moved up a line, not down

Reply:
These were actually unused, so I remove them.